### PR TITLE
[ePD] PCA Template adjustment

### PR DESCRIPTION
**Pull request by roman-karpovich**

### CODE REVIEW

File: src/etools/applications/partners/views/v1.py [hunk 1]
```diff
@@ -29,6 +29,12 @@ class PCAPDFView(LoginRequiredMixin, PDFTemplateView):
         "ifrc_french": "pca/ifrc_french_pdf.html"
     }
 
+    def dispatch(self, request, *args, **kwargs):
+        # increase permissions since we save current user to agreement
+        if not (request.user.is_authenticated and request.user.is_staff):
```
[the remark this answers is not in the document]
Comment (robertavram) [reply src/etools/applications/partners/views/v1.py:34]: this item is pending just in case

**Comment by robertavram** [comment]
1 item pending

File: src/etools/applications/partners/views/v1.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (roman-karpovich) [reply src/etools/applications/partners/views/v1.py]: updated

**Comment by roman-karpovich** [comment]
branch was updated from develop to have tests working

**Pull request merged**